### DISCUSSION TITLE
pro guard query

### DISCUSSION THREAD

**Discussion by mdsaiyad** [trustwallet/wallet-core#1425] [Q&A]
Hi i am using com.trustwallet:wallet-core dependency and its working fine when i am not using proguard but when i use it with proguard it get error.. so i need to know Proguard rules of this library .. PLease let me know waiting for early and positive feedback

**Comment by vikmeup** [comment]
Did you consider to exclude wallet core from pro guard?

**Comment by hewigovens** [comment]
could you please share the error logs? we haven't tried proguard rules
